src/server/player_sao.cpp

```diff
@@ -124,7 +124,7 @@ std::string PlayerSAO::getClientInitializationData(u16 protocol_version)
 		msg_os << serializeLongString(generateUpdateBonePositionCommand(
 			bone_pos.first, bone_pos.second.X, bone_pos.second.Y)); // m_bone_position.size
 	}
-	msg_os << serializeLongString(generateUpdateAttachmentCommand()); // 4
+	msg_os << serializeLongString(generateUpdateAttachmentCommand()); // 5
```

Comment:
Incorrect change. If even - this should be changed to `4 + m_bone_position.size`